src/saveload/saveload.cpp

```diff
@@ -267,7 +267,7 @@
  *  197   27978   1.8.x
  *  198
  *  199
- *  200
+ *  200   Move and extend railtypes
```

Comment:
Nowhere else is a comment about what the version is doing. But it used to have a revision that caused it .. possibly we should have a githash, but we don't know that till it is merged .. PR number? Dunno ..

Reply:
The reason I added it is that with just a number, git will not see a conflict when different branches also increase the version number. Adding the description (indeed, we don't know the githash yet, but PR# code be done) will make other changes conflict instead of "helpfully" being silently merged away.

Reply:
@frosch123 indicates the versions between stables are not relevant; possibly we can just squash it down to '200', removing '198' and '199'? Would that also solve the problem?

Reply:
As far as I know, the reason we've kept adding it is so that it makes it easy to find the correct revision of a savegame bump with git blame.

If we start to add a comment, it also makes conflicting savegame bumps actually conflict in git, but comments shouldn't be added later as that messes up git blame.

Reply:
Agree that including a description of the bump-reason is a good idea. If it can be attributed to a specific PR or ticket number, including that is also appropriate IMO.